packages/react/src/patterns/blocks/LogoGrid/LogoGrid.js

```diff
@@ -18,61 +19,68 @@ const { prefix } = settings;
+ * @param {string} props.heading Heading for the Logo Grid Pattern
  * @param {Array} props.logosGroup Array of object with label, imgSrc and altText properties
+ * @param {object} props.cta CTA object
+ * @param {boolean} props.hideBorder Set to true to hide the grid bottom border
  * @returns {*} Logo Grid Pattern JSX object
  */
-const LogoGrid = ({ theme, title, logosGroup }) => {
+const LogoGrid = ({ heading, logosGroup, cta, hideBorder }) => {
   /**
    * sets the class name based on theme type
    *
    * @param {string} theme theme type ( g10 | white/default )
    * @returns {string} theme css class names
    */
-  const setTheme = theme => {
-    return theme && `${prefix}--logo-grid--${theme}`;
-  };
 
-  return featureFlag(
-    DDS_LOGO_GRID,
+  return (
     <section
-      data-autoid={`${stablePrefix}--logo-grid`}
-      className={classNames(`${prefix}--logo-grid`, setTheme(theme))}>
+      data-autoid={`${stablePrefix}--logo-grid ${prefix}--logo-grid`}
+      className={classNames(`${prefix}--logo-grid`, {
+        [`${prefix}--logo-grid__no-border`]: hideBorder,
+      })}>
       <div className={`${prefix}--logo-grid__container`}>
-        <div className={`${prefix}--logo-grid__row`}>
-          <div className={`${prefix}--logo-grid__col`}>
-            <h3 className={`${prefix}--logo-grid__title`}>{title}</h3>
-          </div>
-        </div>
-        <div className={`${prefix}--logo-grid__row`}>
-          <div className={`${prefix}--logo-grid__col`}>
-            <div className={`${prefix}--logo-grid__wrapper`}>
-              {logosGroup.map(placeholder => (
-                <div
-                  className={`${prefix}--logo-grid__logo`}
-                  key={placeholder.label}>
-                  <img src={placeholder.imgSrc} alt={placeholder.altText} />
+        <div
+          className={`${prefix}--logo-grid__wrapper ${prefix}--grid ${prefix}--grid--full-width`}>
+          <ContentBlock heading={heading} cta={cta}>
+            <div className={`${prefix}--logo-grid__row`}>
+              {logosGroup.map((placeholder, index) => (
+                <div className={`${prefix}--logo-grid__col`} key={index}>
+                  <a
+                    href={placeholder.href}
+                    className={`${prefix}--logo-grid__link`}>
+                    <div
+                      className={`${prefix}--logo-grid__logo`}
+                      key={placeholder.label}>
+                      <Image
+                        defaultSrc={placeholder.imgSrc}
+                        classname={`${prefix}--logo-grid_img`}
+                        alt={placeholder.altText}
+                      />
+                    </div>
+                  </a>
                 </div>
               ))}
             </div>
-          </div>
+          </ContentBlock>
         </div>
       </div>
     </section>
   );
 };
 
 LogoGrid.propTypes = {
-  theme: PropTypes.string,
-  title: PropTypes.string,
+  heading: PropTypes.string,
```

Comment:
Sorry to keep piling on, @asudoh can probably elaborate more but we are moving to use React DocGen and might be a good place here to move to that over JSDocs.

Reply:
Yeah I have’s done JSDoc to React DocGen migration for blocks, but component owners doing it will definitely help the effort (if it’s containable).

Another thing I found recently is that running Jest at top-level directory doesn’t seem to work for correctly updating snapshots. I needed to run it from `react` package.

Reply:
not a problem; reading up on it now.

Reply:
@reesedownes the Doc tab doesn't seem to be rendering properly: https://ibmdotcom-react.s3.us-south.cloud-object-storage.appdomain.cloud/deploy-previews/2489/index.html?path=/docs/patterns-blocks-logogrid--default

Reply:
@jeffchew To show the docs there, we need:

* Rename `README.md` to `README.stories.mdx`
* Update the `import` statement in `*.stories.js` to import the new file name
* Replace the `parameters` section in story with:

```javascript
parameters: {
  ...readme.parameters,
},
```